lib/osf-components/addon/components/registries/registration-form-navigation-dropdown/template.hbs

```diff
@@ -3,70 +3,47 @@
+                    @type='link'
+                    local-class= 'ToggleNav__button'
+                >
                     <FaIcon @icon='times' />
                 </OsfButton>
-            </div>
-            <div local-class='Anchors' data-test-form-block-anchors>
+            </h2>
+            <ol local-class='Anchors' data-test-form-block-anchors>
                 {{#each this.blocksWithAnchor as |block|}}
-                    {{#if (eq block.blockType 'page-heading')}}
-                        <div local-class='PageHeadingAnchor'>
-                            <OsfLink
-                                data-test-page-anchor={{block.elementId}}
-                                data-analytics-name='Go to page: {{block.displayText}}'
-                                @href='#{{block.elementId}}'
-                                @onClick={{action dd.close}}
-                            >
-                                {{block.displayText}}
-                            </OsfLink>
-                        </div>
-                    {{/if}}
-                    {{#if (eq block.blockType 'section-heading')}}
-                        <div local-class='SectionHeadingAnchor'>
-                            <OsfLink
-                                data-test-section-anchor={{block.elementId}}
-                                data-analytics-name='Go to section: {{block.displayText}}'
-                                @href='#{{block.elementId}}'
-                                @onClick={{action dd.close}}>
-                                {{block.displayText}}
-                            </OsfLink>
-                        </div>
-                    {{/if}}
-                    {{#if (eq block.blockType 'subsection-heading')}}
-                        <div local-class='SubsectionHeadingAnchor'>
-                            <OsfLink
-                                data-test-subsection-anchor={{block.elementId}}
-                                data-analytics-name='Go to subsection: {{block.displayText}}'
-                                @href='#{{block.elementId}}'
-                                @onClick={{action dd.close}}>
-                                {{block.displayText}}
-                            </OsfLink>
-                        </div>
-                    {{/if}}
-                    {{#if (eq block.blockType 'question-label')}}
-                        <div local-class='LabelAnchor'>
-                            <OsfLink data-test-label-anchor={{block.elementId}}
-                                data-analytics-name='Go to label: {{block.displayText}}'
-                                @href='#{{block.elementId}}'
-                                @onClick={{action dd.close}}>
-                                {{block.displayText}}
-                            </OsfLink>
-                        </div>
-                    {{/if}}
+                    <li local-class='{{block.blockType}}'>
```

Comment:
generally, we should avoid this sort of coupling -- directly using state as a css class can be confusing and hard to maintain as things change over time.

i'm ok making an exception in this case because it's concise and readable, but i think it's a pattern to be wary of in the future.